connectors/file/translator-parquet/src/main/java/org/teiid/translator/parquet/ParquetUpdateExecution.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags and
+ * the COPYRIGHT.txt file distributed with this work.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.teiid.translator.parquet;
```

Comment:
Are we supporting updates and deletes with Paraquet? typically with file-based we do not support these. In that case this class goes away.

Reply:
There's an ExcelUpdateExecution in excel. Thus I just put a class in there. I don't know yet if these are to be supported in Parquet. You guys can tell better if it should be.

Reply:
There's an ExcelUpdateExecution in excel. Thus I just put a class in there. I don't know yet if these are to be supported in Parquet. You guys can tell better if it should be.